io_scene_niftools/modules/nif_export/scene/__init__.py

```diff
@@ -50,7 +50,18 @@ def get_version_data():
     NifLog.info(f"Writing NIF version 0x{version:08X}")
 
     # get user version and user version 2 for export
-    user_version = b_scene.user_version if b_scene.user_version else b_scene.USER_VERSION.get(game, 0)
-    user_version_2 = b_scene.user_version_2 if b_scene.user_version_2 else b_scene.USER_VERSION_2.get(game, 0)
+    # unfortunately, if USER_VERSION/USER_VERSION_2 is set to 0,
```

Comment:
USER_VERSION and USER_VERSION_2 were removed in https://github.com/niftools/blender_niftools_addon/commit/1ddbe4ff9091ab5c942d6f7cd646441b80380fdf , so they don't need to be checked for. This piece of code is already adjusted in https://github.com/niftools/blender_niftools_addon/pull/591/commits/c8488a1c2326bb3f61d86da14eb2263dd902cfe7 to fix the error you (probably) ran into. You could rebase to that fork to not need these changes.

Reply:
By the way, I decided just to remove this change.  When will c8488a1 get into master?

Reply:
When Hendrix has approved #591 .